### ISSUE TITLE
Issue posted on course discussion

### ISSUE THREAD

**Issue by coryhouse** [joshbula/LearningReact#1]
Hi Josh - I got an email notification that you posted on the course discussion, but I don't see your comment on the course discussion board. Did you delete? If not, can you see it? (Pluralsight has had some reliability problems with the discussion board).

Oh, and if your issue persists, I'll give it a look. Please just reply here with your issue.

**Comment by joshbula** [comment]
Thanks Cory, I did delete it because I figured out my problems after going
back a few videos and re-watching them. I just forgot to add authors to the
root reducer.

Also, updating CourseForm.js to use 'prop-types' I had to remove the
"React." part of all the "React.PropTypes.object.isRequired" statements so
they just say "PropTypes.object.isRequired"  That might be something to add
in the "Update course exercise" pull request that you shared.

Thanks again for the great course and for being so responsive to discussion
board!

...Josh
